documentation/doc-Migration_Toolkit_for_Virtualization/modules/creating-migration-plan.adoc

```diff
@@ -12,15 +12,15 @@ A migration plan allows you to group virtual machines that should be migrated to
 .Prerequisites
 
 * You must create a VDDK image and add it to the `v2v-vmware` config map.
-* You must add source and target providers to the web console.
```

Comment:
I think here we can elaborate that the current target provider is added by default during MTV installation, and if the target provider is not the one from which we run the MTV migration, then this other target provider should be added.
Suggested text:
"If the target provider is the current cluster, then it's added duting MTV installation automatically. However if the target provider is another cluster, then it must be added"

Reply:
Done